dali/test/python/torchvision/test_tv_centercrop.py

```diff
@@ -0,0 +1,128 @@
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Sequence, Union, Literal
+
+import torch
+import numpy as np
+
+from nose2.tools import params
+from nose_utils import assert_raises
+from nvidia.dali.experimental.torchvision import Compose, CenterCrop
+import torchvision.transforms.v2 as transforms
+
+
+def build_centercrop_transform(
+    size: Union[int, Sequence[int]], batch_size: int = 1, device: Literal["cpu", "gpu"] = "cpu"
```

Comment:
Nitpick: I think we can use the `|` syntax already.

`size: int | Sequence[int]`